### PR TITLE
API update pool

### PR DESCRIPTION
**Pull request by ahalekelly**
Inspired by @TrueMilli #503

pool: return the pool dict. In the cast of binary markets, this contains the number of Yes and No shares in the liquidity pool, and in the case of free response markets, this contains the total value invested in each of the answers. Previously the pool value returned the sum of the pool shares for binary markets and null for free response markets, so this is a breaking change but provides more info and I might be the only one using this value currently.

totalLiquidity: for binary markets this is unchanged, for free response markets it now returns the total money invested in the market. Used @TrueMilli's function

But it feels like CPMM and DPM markets are fundamentally different enough that I'm not sure it makes sense to combine them like this. DPM markets don't exactly have liquidity like CPMM markets do, I would rather keep totalLiquidity as CPMM only and make a new variable totalValue that is the total amount invested in the market. If we do this, we should make sure it would support #535 as well, which I'm not quite sure how to do yet

### CODE REVIEW

**Comment by ahalekelly** [comment]
Reverted the totalLiquidity changes, #554 and #535 made me more convinced on the need for a totalValue variable, I think we should implement that instead and expose that via the API.

**Comment by ahalekelly** [comment]
Also I think the Pool value on the homepage should show totalValue, the amount up for grabs for the winner.

File: docs/docs/api.md [hunk 1]
```diff
@@ -96,7 +96,8 @@ Requires no authorization.
     tags: string[]
     url: string
 
-    pool: number
+    pool: { outcome: number } // For binary markets, the number of shares in the liquidity pool. For free response markets, the amount of mana invested in each answer
+    totalLiquidyt?: number // Binary markets only, the amount of mana deposited into the liquidity pool
```
Comment (mqp) [inline docs/docs/api.md:100]: Typo here.

**Review by mqp — APPROVED** [review]
Makes total sense to me.

File: docs/docs/api.md (hunk 1 above)
Comment (ahalekelly) [reply docs/docs/api.md:100]: Oh thanks

**Comment by ahalekelly** [comment]
Aw shit, I hate git

**Comment by mqp** [comment]
I love git.

**Pull request merged**